lti_consumer/lti_1p3/key_handlers.py

```diff
@@ -86,13 +86,6 @@ def _get_keyset(self, kid=None):
                 raise exceptions.NoSuitableKeys() from err
             keyset.extend(keys.keys)
 
-        if self.public_key and kid:
```

Comment:
I removed this as the `_get_keyset` function was never being called with a non-none `kid` value. The comment also suggests this was related to `pyjwkest` verification, but verification is passing using `pyjwt`